composer.json

```diff
@@ -40,7 +40,8 @@
             "composer/installers": true,
             "cweagans/composer-patches": true,
             "drupal/core-composer-scaffold": true,
-            "dealerdirect/phpcodesniffer-composer-installer": true
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "phpstan/extension-installer": true
```

Comment:
`phpstan/extension-installer` comes from `drupal/core-dev@^10`

Reply:
We should probably create a 10x branch here